src/message.rs

```diff
@@ -1837,7 +1839,7 @@ async fn ndn_maybe_add_info_msg(
     chat_id: ChatId,
     chat_type: Chattype,
 ) -> anyhow::Result<()> {
-    if chat_type == Chattype::Group {
+    if chat_type == Chattype::Group || chat_type == Chattype::Mailinglist {
```

Comment:
1. A `match` is better.
2. This does not make sense here, `ndn_maybe_add_info_msg` is about the case when delivery to the group failed. If we get an NDN for the mailing list, better write a `warn` and do nothing.
3. Sidenote: have never seen this status message added, not sure if it works properly.